public/javascripts/data/paragraph.js

```diff
@@ -0,0 +1,32 @@
+const Database = require('better-sqlite3');
+
+
+// Function writes paragraph to the database
+exports.insertParagraph = (paragraph) => {
+
+    // TODO:
+    // 1. (user_id, stories_id) should contain current user and current story
+    const db = new Database('controllers/stories.db');
+    const new_paragraph = db.prepare('INSERT INTO paragraphs (user_id, stories_id, content) VALUES (?, ?, ?)');
+    const values = new_paragraph.run(4, 1, paragraph);
+    db.close();
+    return;
+};
+
+// Function gets all paragraphs in the database, stores them in array
+exports.displayAllParagraphs = () => {
+    
+    // TODO:
+    // 1. (stories_id) should contain the current story_id where the POST request was sent from
+    const db = new Database('controllers/stories.db');
+    const all_paragraphs = db.prepare('SELECT content FROM paragraphs WHERE stories_id=1');
+    let all_p_obj = all_paragraphs.all();
```

Comment:
What does `all_paragraphs.all()` return? An array of rows? You might not need to build your own array and can just return the result of `all_paragraphs.all()`